Proxy/HTTP/WhoSpyOnMe?.bambda

```diff
@@ -0,0 +1,49 @@
+/**
+ * WhoSpyOnMe: Burp Suite Bambda for Identifying Tracking Services
+ * FilterOut Burp Suite history to detect and analyze tracking services from web requests
+ * Capable of filtering based on Hosts, endpoints and parameters for detailed insights
+ * Useful for privacy audits and identifying potential data privacy concerns
+ * Future extension plan: 'DontSpyOnMe' to automatically block identified tracking hosts
+ * Contributions welcome: Feel free to contribute and add more hosts for detection
+ * Author: Tur24Tur / BugBountyzip (https://github.com/BugBountyzip)
```

Comment:
Please can you change `Author` to `@author`

Reply:
Thank you , you are right